Practice_02-display/lcd_driver/fb_lcd.h

```diff
@@ -0,0 +1,37 @@
+#ifndef FB_LCD_H_
+#define FB_LCD_H_
+
+#include <linux/init.h>
+#include <linux/module.h>
+#include <linux/device.h>
+#include <linux/err.h>
+#include <linux/kobject.h>
+#include <linux/sysfs.h>
+
+#include <linux/spinlock.h>
+#include <linux/mutex.h>
+#include <linux/of.h>
+#include <linux/of_platform.h>
+#include <linux/platform_device.h>
+#include <linux/gpio.h>
+#include <linux/delay.h>
+#include <linux/spi/spi.h>
+#include <linux/fb.h>
+#include <linux/uaccess.h>
+
+static struct fb_fix_screeninfo st7735sfb_fix = {
+	.id     	= "ST7735S",
+	.type       = FB_TYPE_PACKED_PIXELS,
+	.visual     = FB_VISUAL_TRUECOLOR,
+	.xpanstep   = 0,
+	.ypanstep   = 0,
+	.ywrapstep  = 0,
+	.accel      = FB_ACCEL_NONE,
+};
+
+
+static struct fb_var_screeninfo st7735sfb_var = {
+	.bits_per_pixel = 16,
+};
```

Comment:
Variable definition in header file is mistake. The headers should contain only declarations.